tests/ops/op_math/test_pow_op.py

```diff
@@ -317,6 +318,26 @@ def test_has_matrix_false(self, power_method):
 
         assert op.has_matrix is False
 
+    @pytest.mark.parametrize("z", [-2, 3, 2])
```

Comment:
then we can add an extra parametrization here 
```suggestion
    @pytest.mark.parametrize("z", [-2, 3, 2, 3.0])
```